cms/static/sass/elements/_footer.scss

```diff
@@ -58,5 +62,42 @@
         }
       }
     }
+
+    .footer-content-secondary {
+      @include clearfix();
+      margin-top: $baseline;
+    }
+
+    .footer-about-copyright, .footer-about-openedx {
+      display: inline-block;
+      vertical-align: middle;
+    }
+
+    // platform trademarks
+    .footer-about-copyright {
+      width: flex-grid(4, 12);
+      @include float(left);
+      margin-right: flex-gutter(2);
```

Comment:
oh wait -heh yeah the margin there is a bit wonky. I'll add the @include and see what happens